src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -150,18 +143,12 @@ private void InitManifestModule()
 
         #region DefineDynamicAssembly
 
-        /// <summary>
-        /// If an AssemblyName has a public key specified, the assembly is assumed
-        /// to have a strong name and a hash will be computed when the assembly
-        /// is saved.
-        /// </summary>
         [DynamicSecurityMethod] // Methods containing StackCrawlMark local var has to be marked DynamicSecurityMethod.
```

Comment:
`DynamicSecurityMethod` no longer needed here?

Reply:
It is needed to make `Assembly.GetCallingAssembly` reliable. I will update the comment.